packages/components/card/_index.scss

```
@@ -278,20 +286,25 @@ $card-border-hover-color: $color_nhsuk-grey-3;
 /// Card primary
 
 .nhsuk-card__content--primary {
-  padding-right: 75px;
+  $icon-width: 27px;
+  $icon-height: 27px;
+
+  @include nhsuk-responsive-spacing(5, "padding", $direction: "right", $adjustment: $icon-width + nhsuk-spacing(4));
 
   @include nhsuk-media-query($from: desktop) {
     height: 100%;
   }
 
   .nhsuk-icon {
     display: block;
+    width: $icon-width;
+    height: $icon-height;
     fill: $color_nhsuk-blue;
-    margin-top: -(nhsuk-spacing(2));
+    margin-top: math.floor(math.div($icon-height, 2)) * -1;
```

Is this clearer? I’ve always found that multiplying by -1 to be a slightly unintuitive (but clever!) maths trick to make a number negative...

```suggestion
    margin-top: -(math.floor(math.div($icon-height, 2)));
```